### PR TITLE
kubectl print ingress add column labels

### PR DESCRIPTION
**Pull request by adohe-zz**
Please refer #20291 for more detail.

### CODE REVIEW

**Comment by AdoHe** [comment]
Also when I run `hack/test-go.sh` I noticed there are some errors.

**Comment by bgrant0607** [comment]
cc @kubernetes/kubectl

File: pkg/kubectl/resource_printer.go [hunk 1]
```diff
@@ -880,13 +880,16 @@ func printIngress(ingress *extensions.Ingress, w io.Writer, options PrintOptions
 		}
 	}
 
-	if _, err := fmt.Fprintf(w, "%s\t%v\t%v\t%v\n",
+	if _, err := fmt.Fprintf(w, "%s\t%v\t%v\t%v",
```
Comment (bprashanth) [inline pkg/kubectl/resource_printer.go]: A strange side-effect of appendLabels is that it always appends a new line. Hrm. I guess it works out in this scenario.

**Comment by bprashanth** [comment]
LGTM

File: pkg/kubectl/resource_printer.go (hunk 1 above)
Comment (AdoHe) [inline pkg/kubectl/resource_printer.go]: Originally this method is always called to append user specify columns at last, so a new line delimiter is necessary.

**Comment by bgrant0607** [comment]
```
# k8s.io/kubernetes/pkg/kubectl
_output/local/go/src/k8s.io/kubernetes/pkg/kubectl/resource_printer_test.go:672: cannot use []string literal (type []string) as type bool in field value
_output/local/go/src/k8s.io/kubernetes/pkg/kubectl/resource_printer_test.go:672: too few values in struct initializer
```

**Comment by AdoHe** [comment]
I will fix this.

**Comment by bgrant0607** [comment]
@AdoHe Thanks. I suggest rebasing, just in case.

**Comment by AdoHe** [comment]
I just fix the test issue, also rebase this.

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please github issue: #21188

**Pull request merged**